scripts/source-maintenance.sh

```diff
@@ -378,7 +378,7 @@ do
 	echo -n "	${f}"
 done
 echo " "
-)| sed s%src/%%g >src/tests/Stub.list
+)| sed s%src/%%g >src/tests/Stub.am
```

Comment:
```suggestion
) | sed s%src/%%g > src/tests/Stub.am
```

I do not insist on this "(arguably) clearer/safer formatting" change.